README.md

```diff
@@ -28,6 +28,7 @@ The following example shows how to use the [DRUM](https://github.com/datarobot/d
 4. cd into the repo: `cd datarobot-user-models`
 5. Install the required dependencies: `pip install -r public_dropin_environments/python3_sklearn/requirements.txt`
 6. Install datarobot-drum: `pip install datarobot-drum`
+   1. If you want to install the dev environment, instead `pip install -e custom_model_runner/`
```

Comment:
weird to have a 1. here

Reply:
the indentation grabs my attention as a sub-point of the main point.  if not this, what would you suggest?

Reply:
a bullet?

Reply:
unfortunately no.  i tried, but it's not legit md